plugins/repository-gcs/src/main/java/org/elasticsearch/repositories/gcs/GoogleCloudStorageBlobStore.java

```diff
@@ -231,14 +228,9 @@ void writeBlob(String blobName, BytesReference bytes, boolean failIfAlreadyExist
             // Compute md5 here so #writeBlobResumable forces the integrity check on the resumable upload.
```

Comment:
This comment seems out of place now. Perhaps the MessageDigests.digest call could still be done into a local next to this comment?

Reply:
++